### DISCUSSION TITLE
How to create empty containers?

### DISCUSSION THREAD

**Discussion by dsech** [terrastruct/d2#1114] [Q&A] [answered]
I was wondering if it's possible to create empty containers. For example here, "a", "b", and "c" are siblings, but with different levels of details:

https://play.d2lang.com/?script=SrRSqOZSUMjNTynNSYWwFRSKS5MgAlwKCrVctVxcSciqQALJXFyJCrp2CkkgIpkLEAAA__8%3D&sketch=1&theme=300&layout=dagre&

I'm curious if it's possible to make "c" look the same as "a" and "b" (same border and background), even though there are no subcomponents defined inside.

Thank you!

**Comment by alixander** [comment, marked as answer]
@dsech Oh hmmm. two options:

1. Style it the same: https://play.d2lang.com/?script=VJExCsMwDEV3neKRzgUnU9HQLblHnGRrCdT2VHz3YokWdxHiIfE_vFV5CzzPvTwO3yGV6ECgShWJ_VUDmwPr8P36E6jkVzmM996U_czph5XhMs_LuNwGiwIrpUwhCLgrZZxCC1253oltbPIJAAD__w%3D%3D&sketch=1&theme=300&layout=dagre&

2. Put a hidden child inside: https://play.d2lang.com/?script=SrRSqOZSUMjNTynNSYWwFRSKS5MgAlwKCrVctVxcSciqQALJEIGMzJSU1Dw9sFP0oKFipWAAUpGooGunkAQikrkAAQAA__8%3D&sketch=1&theme=300&layout=dagre&

**Reply by alixander** [reply]
It's an interesting point I hadn't thought of though. Will think about this more

**Reply by dsech** [reply]
For me the hidden child is a really good workaround right now. 
And I really love it because there's no need for complicated styling on the parent container.

Cheers!
